api_app/classes.py

```diff
@@ -121,9 +123,23 @@ def after_run(self):
         self.report.save()
 
     def after_run_success(self, content: typing.Any):
+        # exhaust generator
         if isinstance(content, typing.Generator):
             content = list(content)
-        self.report.report = content
+        # avoiding JSON serialization errors for types: File and bytes
+        report_content = content
+        if isinstance(report_content, typing.List):
+            if all(isinstance(n, File) for n in report_content):
+                report_content = [
+                    base64.b64encode(f.read()).decode("utf-8") for f in report_content
+                ]
+            if all(isinstance(n, bytes) for n in report_content):
+                report_content = [
+                    base64.b64encode(b).decode("utf-8") for b in report_content
+                ]
+
+        self.content = content
```

Comment:
not necessary

Reply:
I need the content inside `ingestor_manager/classes.py`

```python
    def after_run_success(self, content):
        super().after_run_success(content)
        self._config: IngestorConfig
        deque(
            self._config.create_jobs(
                # every job created from an ingestor
                self.content,
                TLP.CLEAR.value,
                self._user,
                self._config.delay,
            ),
            maxlen=0,
        )
```